tictactoe.py

```diff
@@ -121,10 +172,18 @@ def checkWin(gameBoard):
         if playerWin is True:
             break
 
-        userO = input("User O. Enter position number: ")
+        aiMove()
 
-        checkPlay(userO, OGAMEPIECE)
         playerWin = checkWin(gameBoard)
         displayBoard()
         if playerWin is True:
             break
+
+
+print("\n      Tic Tac Toe")
+displayBoard()
+gameType = input("(1) player or (2) player? Enter number: ")
+if gameType == "2":
```

Comment:
this is very silly and not really anything critical but I think it is very funny than 2 player game runs the game1 function and 1 player game runs the game2 function. You could probably rename them to singlePlayer() and multiPlayer() or something if you really wanted to